Video.js

```diff
@@ -234,6 +240,7 @@ export default class Video extends Component {
       onVideoSeek: this._onSeek,
       onVideoEnd: this._onEnd,
       onVideoBuffer: this._onBuffer,
+      onBandwidthUpdate: this._onBandwidthUpdate,
```

Comment:
Please change this to onVideoBandwidthUpdate. The outward facing prop can still be called onBandwidthUpdate. Event names are global across the app, so if another module (e.g. audio) also declares onBandwidthUpdate, it will conflict.

You can copy the pattern for other events such as onVideoProgress -> onProgress